<ui version="4.0" >
 <class>NuevoClienteDialogClass</class>
 <widget class="QDialog" name="NuevoClienteDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Nuevo Cliente</string>
  </property>
  <property name="windowIcon" >
   <iconset>images/NCliente.PNG</iconset>
  </property>
  <widget class="QWidget" name="gridLayout" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Apellidos</string>
      </property>
     </widget>
    </item>
        <item row="0" column="1" >
     <widget class="QLineEdit" name="lineEdit_4" />
    </item>
    <item row="0" column="3" colspan="2" >
     <widget class="QLineEdit" name="lineEdit_5" />
    </item>
    <item row="1" column="1" colspan="4" >
     <widget class="QLineEdit" name="lineEdit" />
    </item>
    <item row="2" column="1" colspan="4" >
     <widget class="QLineEdit" name="lineEdit_2" />
    </item>
    <item row="0" column="0" >
     <widget class="QLabel" name="label_4" >
      <property name="text" >
       <string>Codigo</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Direccion</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4" >
     <widget class="QLineEdit" name="lineEdit_3" />
    </item>
        <item row="4" column="1" colspan="4" >
     <widget class="QLineEdit" name="lineEdit_6" />
    </item>
        <item row="5" column="4" >
     <widget class="QPushButton" name="pushButton" >
      <property name="text" >
       <string>AÃ±adir</string>
      </property>
     </widget>
    </item>
    
        <item rowspan="0" row="7" column="0" colspan="2" >
     <widget class="QTableWidget" name="tableWidget" />
    </item>
    <item rowspan="3" row="5" column="0" colspan="4" >
     <widget class="QTableWidget" name="tabla" >
      <column>
       <property name="text" >
        <string>Telefono</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="6" column="4" >
     <widget class="QPushButton" name="pushButton_2" >
      <property name="text" >
       <string>Borrar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" >
     <widget class="QLabel" name="label_5" >
      <property name="text" >
       <string>NIF</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="5" >
     <layout class="QHBoxLayout" >
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelarButton" >
        <property name="text" >
         <string>&amp;Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guardarButton" >
        <property name="text" >
         <string>&amp;Guardar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label_6" >
      <property name="text" >
       <string>Poblacion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>cancelarButton</sender>
   <signal>clicked()</signal>
   <receiver>NuevoClienteDialogClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>264</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel" >
     <x>189</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
